server/internal/virtual/components.go

```diff
@@ -118,6 +118,7 @@ func initComponents(
 		wmLogger   *log.WatermillLogAdapter
 		publisher  message.Publisher
 		subscriber message.Subscriber
+		APIWrapper *api.RunnerWrapper
```

Comment:
`APIWrapper` does not connected to watermill, don't declarate it here. You can just do `APIWrapper := api.NewWrapper(API, AdminAPIRunner)`